src/jit/compatjit/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+project(compatjit)
+
+# This compatjit.dll is only built if we are not building JIT32 as compatjit.dll.
```

Comment:
Yes. They're not quite the same, since one is an altjit and one is not. But essentially they are the same.